<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowGraspQuality</class>
 <widget class="QMainWindow" name="MainWindowGraspQuality">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraspStudio - GraspQuality</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>81</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Close EEF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonOpen">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>81</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Open EEF</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>510</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Visualization</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxColModel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>540</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Collision Model</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxEEF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>80</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select end effector</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>181</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Object</string>
       </property>
       <widget class="QSlider" name="horizontalSliderRo">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-50</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>80</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Move Object</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderZ">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-50</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderX">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-50</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>100</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ro</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>120</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Pi</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderPi">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-50</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>140</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ya</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>60</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderYa">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-50</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>40</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderY">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-50</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonToTCP">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>to TCP</string>
        </property>
       </widget>
      </widget>
      <widget class="QCheckBox" name="checkBoxCones">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>560</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Friction Cones</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonQuality">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Grasp Quality</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxGWS1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>580</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show GWS (force)</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxGWS2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>600</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show GWS (torque)</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxOWS2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>640</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show OWS (torque)</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxOWS1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>620</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show OWS (force)</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
